src/Controls/GlobeControls.js

```diff
@@ -759,26 +770,36 @@ class GlobeControls extends THREE.EventDispatcher {
         }
     }
 
-    handleZoom(event) {
+    handleZoom(event, dollyOrigin) {
```

Comment:
I can't remember if there's a change of state for the zoom?
to handle the state zoom

Reply:
no state change for the zoom. I need to disguish between a regular zoom and a dolly one. Maybe I could 
```suggestion
    handleZoom(event, dollyFlag) {
```
with `dollyFlag` used to know if I need to update `pickedPosition` value

Reply:
> no state change for the zoom. I need to disguish between a regular zoom and a dolly one. Maybe I could

This would seem to confirm that `StateControl` should manage its two states?

Reply:
in my last commit, i've used directly the event type to distinguish between the 2 zoom